airbyte-webapp/src/views/layout/SideBar/SideBar.tsx

```diff
@@ -129,12 +129,12 @@ const SideBar: React.FC = () => {
         <li>
           <SidebarPopout options={[{ value: "docs" }, { value: "slack" }, { value: "recipes" }]}>
             {({ onOpen, isOpen }) => (
-              <div className={getPopoutStyles(isOpen)} onClick={onOpen}>
+              <button className={getPopoutStyles(isOpen)} onClick={onOpen} tabIndex={0}>
```

Comment:
The `tabIndex` should not be needed here. `button` elements are focusable by default. (Likely a left over from some in between state?)